Test-NETHLK/LICENSE.md

```diff
@@ -0,0 +1,23 @@
+# License for the contents of the archive Test-NETHLK.2021.7.28.55.zip

```

Comment:
Can't you download the license from github together with the script?

Reply:
The license is already present in the repository and in the module archive itself. I just thought that I should put a copy into our repository.

Reply:
I don't think it is needed in such a case. We are downloading the module archive with the license.

Reply:
Removed